### PR TITLE
gnome-video-effects: 0.4.3 → 0.5.0

### PR DESCRIPTION
**Pull request by jtojnar**
https://gitlab.gnome.org/GNOME/gnome-video-effects/blob/0.5.0/NEWS#L4-6

### CODE REVIEW

File: pkgs/development/libraries/gnome-video-effects/default.nix [hunk 1]
```diff
@@ -1,20 +1,20 @@
-{ stdenv, fetchurl, pkgconfig, intltool, gnome3 }:
-let
+{ stdenv, fetchurl, pkgconfig, meson, ninja, gettext, gnome3 }:
```
Comment (worldofpeace) [inline pkgs/development/libraries/gnome-video-effects/default.nix]: Can we have one per line?

File: pkgs/development/libraries/gnome-video-effects/default.nix [hunk 2]
```diff
@@ -1,20 +1,20 @@
...
 
   src = fetchurl {
-    url = "mirror://gnome/sources/${pname}/${stdenv.lib.versions.majorMinor version}/${name}.tar.xz";
-    sha256 = "06c2f1kihyhawap1s3zg5w7q7fypsybkp7xry4hxkdz4mpsy0zjs";
+    url = "mirror://gnome/sources/${pname}/${stdenv.lib.versions.majorMinor version}/${pname}-${version}.tar.xz";
+    sha256 = "1j6h98whgkcxrh30bwvnxvyqxrxchgpdgqhl0j71xz7x72dqxijd";
   };
 
-  nativeBuildInputs = [ pkgconfig intltool ];
+  nativeBuildInputs = [ pkgconfig meson ninja gettext ];
```
Comment (worldofpeace) [inline pkgs/development/libraries/gnome-video-effects/default.nix]: same as https://github.com/NixOS/nixpkgs/pull/66934/commits/9df736b4910a030743c0d9528998399e8b3eca76#r315306373 style

**Comment by worldofpeace** [comment]
Huh, from how the substitute the .pc file in meson `datarootdir` isn't relative to `${prefix}` anymore
```diff
--- ./result/share/pkgconfig/gnome-video-effects.pc	1969-12-31 19:00:01.000000000 -0500
+++ /nix/store/27ml1s87l109aircbf47aigjcqsjd346-gnome-video-effects-0.5.0/share/pkgconfig/gnome-video-effects.pc	1969-12-31 19:00:01.000000000 -0500
@@ -1,7 +1,7 @@
-prefix = /nix/store/rxg02n90vc3y4nqjl296lkmrpamjik4q-gnome-video-effects-0.4.3
-datarootdir = ${prefix}/share
+prefix = /nix/store/27ml1s87l109aircbf47aigjcqsjd346-gnome-video-effects-0.5.0
+datarootdir = /nix/store/27ml1s87l109aircbf47aigjcqsjd346-gnome-video-effects-0.5.0/share/gnome-video-effects
 effectsdir = ${datarootdir}/gnome-video-effects
 
 Name: gnome-video-effects
 Description: A collection of GStreamer effects to be used in different GNOME Modules
-Version: 0.4.3
+Version: 0.5.0

```

and it looks like `effectsdir` will be a non-existent path?

**Comment by jtojnar** [comment]
Hmm, you are right. Cheese relies on `XDG_DATA_DIRS` instead of the `.pc` file so that’s why the effects show up in there.

**Comment by jtojnar** [comment]
Before I managed to rebase the upstream MR so that the patch is applicable, it got merged :man_shrugging: So I added it in-tree 😦

**Comment by worldofpeace** [comment]
> Before I managed to rebase the upstream MR so that the patch is applicable, it got merged man_shrugging So I added it in-tree frowning

:rofl: Wish some other MR's would get merged that fast. It's a pretty small patch so I guess it's fine.

**Review by worldofpeace — APPROVED** [review]
LGTM.

**Pull request merged**